doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1097 @@
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_210_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Added:
+
+  * Information about :ref:`gs_supported_OS` to the :ref:`gs_recommended_versions` page.
+  * Information about :ref:`app_build_output_files` on the :ref:`app_build_system` page.
+  * Information about :ref:`gs_debugging` on the :ref:`gs_testing` page.
+    Also added links to this section in different areas of documentation.
+  * An option to configure IEEE 802.15.4 ACK frame timeout at build time using :kconfig:option:`CONFIG_NRF_802154_ACK_TIMEOUT_CUSTOM_US`.
+  * Serial recovery of the image of nRF5340's ``cpunet`` build targets even when the simultaneous cores upgrade is disabled (:kconfig:option:`CONFIG_NRF_MULTI_IMAGE_UPDATE` set to ``n``).
+    This is enabled with the :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE`.
+
+* Updated :ref:`app_memory` page with sections about Gazell and NFC.
+
+RF Front-End Modules
+--------------------
+
+* Added:
+
+  * The :kconfig:option:`CONFIG_MPSL_FEM_ONLY` Kconfig option that allows the :ref:`nrfxlib:mpsl_fem` API to be used without other MPSL features.
+    The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
+    You cannot use other MPSL features when this option is enabled.
+  * The possibility to add custom models which split the requested TX power between the power on SoC output and the FEM gain in a way desired by the user.
+
+* Fixed a build error that occurred when building an application for nRF53 SoCs with Simple GPIO Front-End Module support enabled.
+
+Wi-Fi
+-----
+
+* Added experimental support for the nRF7002 DK that includes the nRF7002 companion IC.
+  For more information, see the :ref:`ug_nrf70` guide.
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in your application.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for changing the radio transmitter's default power level using the :c:func:`sdc_default_tx_power_set` function.
+  * Support for changing the peripheral latency mode using the :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set` function.
+  * Support for changing the default TX power using Kconfig options that start with ``CONFIG_BT_CTLR_TX_PWR_*``.
+
+* Updated LTE dual-chip Coex support on the nRF52 Series.
+  It is now ready for production.
```

Comment:
Can we also get this into the highlight list, since it gets production supported?